### PR TITLE
chore: Add deployment scripts for Volta and EWC

### PR DESCRIPTION
**Pull request by josipbagaric**
- Adds HardHat network definitions for Volta and EWC
- Adds deployment scripts

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
-import '@nomiclabs/hardhat-ethers'
-import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat';
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-waffle';
+
+// THIS PRIVATE KEY SHOULD ONLY BE USED IN TEST ENVIRONMENTS
+// It has been leaked publicly and its funds can be stolen at any time
+// Replace with your own private key
+const deployKey = "0xd9066ff9f753a1898709b568119055660a77d9aae4d7a4ad677b8fb3d2a571e5";
```
Comment (drgorb) [inline hardhat.config.ts]: I prefer when private keys are imported from a file that is ignored. Especially as this is meant to be a template repo, we should encourage practices that have less risk of leaking keys.

File: hardhat.config.ts [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
   solidity: "0.8.4",
+  networks: {
+    volta: {
+      url: "https://volta-rpc.energyweb.org",
+      accounts: [deployKey],
+      chainId: 73799
+    },
+    ewc: {
+      url: "https://rpc.energyweb.org",
+      accounts: [deployKey],
```
Comment (drgorb) [inline hardhat.config.ts:27]: an alternative to the above comment would be to import the key only for EWC deployments by renaming the `deployKey` to something like `deployKeyEWC` and importing this one from an external file if it exists

File: scripts/deploy.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  
+    console.log("KycToken deployed to:", kycToken.address);
+}
+  
+main()
+    .then(() => process.exit(0))
+    .catch((error) => {
+        console.error(error);
+        process.exit(1);
+    });
```
Comment (drgorb) [inline scripts/deploy.js]: missing newline at the end of file

**Review by drgorb — APPROVED** [review]
this looks great, I'm only a bit worried by the practice of putting a private key in a file which is committed to the repo. This could lead to problems in the future if a developer is being hasty.

But this change can be added in a future release, hence the approval

**Comment by josipbagaric** [comment]
@drgorb 100% agree. Moved it to a `.env` file!

Please merge it if all looks good (I don't have the permissions).

**Review by drgorb — APPROVED** [review]

**Pull request merged**